src/batch.rs

```diff
@@ -431,6 +433,15 @@ fn ddbjson_val_to_attrval(ddb_jsonval: &JsonValue) -> Option<AttributeValue> {
             n: Some(x.as_str().unwrap().to_string()),
             ..Default::default()
         })
+    } else if let Some(x) = ddb_jsonval.get("B") {
+        Some(AttributeValue {
+            b: Some(Bytes::from(
+                general_purpose::STANDARD
```

Comment:
[STANDARD_NO_PAD](https://docs.rs/base64/latest/base64/engine/general_purpose/constant.STANDARD_NO_PAD.html#) accepts more wide range of encoded characters. I think tolerant decoding is preferable.

Reply:
I believe the `general_purpose::STANDARD` (with padding version) is suitable for this for 2 reasons.

1.  the document linked below uses base64 with padding.
https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.NamingRulesDataTypes.html#HowItWorks.DataTypes.SetTypes
```
["U3Vubnk=", "UmFpbnk=", "U25vd3k="]
```

2.  I tried to batch-write a binary without padding using AWS CLI, but it does not accept base64 without padding:

```
$ aws --version
aws-cli/2.13.24 Python/3.11.5 Linux/6.2.0-1013-aws exe/x86_64.ubuntu.22 prompt/off

$ aws dynamodb batch-write-item --request-items file://tests/resources/test_batch_write.json 

Invalid base64: "U3Vubnk"
```

Reply:
Thank you for confirmation. It makes sense. Please go ahead with `general_purpose::STANDARD`.